src/Ionide.Analyzers/Hints/CopyAndUpdateRecordChangesAllFieldsAnalyzer.fs

```diff
@@ -0,0 +1,65 @@
+﻿module Ionide.Analyzers.Hints.CopyAndUpdateRecordChangesAllFieldsAnalyzer
+
+open FSharp.Analyzers.SDK
+open FSharp.Analyzers.SDK.ASTCollecting
+open FSharp.Analyzers.SDK.TASTCollecting
+open FSharp.Compiler.Symbols
+open FSharp.Compiler.Text
+open FSharp.Compiler.Syntax
+
+type UpdateRecord = SynExprRecordField list * range
+
+[<CliAnalyzer("CopyAndUpdateRecordChangesAllFieldsAnalyzer",
+              "Detect if all fields of an update record are updated.",
```

Comment:
This reads a bit rough to me. Maybe something like
- "Detect if all fields in a record update expression are updated."
- "Detect if all fields in an updated record are updated."